### PR TITLE
test/e2e/framework move func logClusterImageSources to test/e2e/e2e.go

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
 
 /kind cleanup
 
**What this PR does / why we need it**:

LogClusterImageSources is only used in test/e2e/e2e.go , so move it to there.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/framework/BUILD [hunk 1]
```diff
@@ -8,7 +8,6 @@ go_library(
         "expect.go",
         "flake_reporting_util.go",
         "framework.go",
-        "google_compute.go",
```
Comment (tanjunchen) [inline test/e2e/framework/BUILD]: attention:
This function is in test/e2e/framework/google_compute.go

**Review by oomichi — COMMENTED** [review]
/lgtm

**Review by gavinfish — COMMENTED** [review]
/lgtm

**Pull request merged**